dev-db/postgresql/postgresql-9.6.10.recipe

```diff
@@ -0,0 +1,160 @@
+SUMMARY="A powerful object-relational database system"
+DESCRIPTION="PostgreSQL has more than 15 years of active development and a \
+proven architecture that has earned it a strong reputation for reliability, \
+data integrity, and correctness.
+
+* It runs on all major operating systems, including Linux, UNIX (AIX, BSD, \
+HP-UX, SGI IRIX, Mac OS X, Solaris, Tru64), and Windows.
+* It is fully ACID compliant, has full support for foreign keys, joins, \
+views, triggers, and stored procedures (in multiple languages).
+* It includes most SQL:2008 data types, including INTEGER, NUMERIC, BOOLEAN, \
+CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
+* It supports storage of binary large objects, including pictures, sounds, or \
+video.
+* It has native programming interfaces for C/C++, Java, .Net, Perl, Python, \
+Ruby, Tcl, ODBC, among others, and exceptional documentation."
+HOMEPAGE="https://www.postgresql.org/"
+COPYRIGHT="1996-2017 PostgreSQL Global Development Group"
+LICENSE="PostgreSQL"
+REVISION="1"
+SOURCE_URI="http://ftp.postgresql.org/pub/source/v$portVersion/postgresql-$portVersion.tar.bz2"
+CHECKSUM_SHA256="8615acc56646401f0ede97a767dfd27ce07a8ae9c952afdb57163b7234fe8426"
+SOURCE_DIR="postgresql-$portVersion"
+PATCHES="postgresql-$portVersion.patch"
```

Comment:
You should probably rename the patch and:
```diff
-PATCHES="postgresql-$portVersion.patch"
+PATCHES="postgresql-$portVersion.patchset"
```
After all, even if you did not use `haikuporter -e` to create the patch, it is compatible with haikuporter's patchset format.

BTW, instead of renaming the patch manually, you know you can have haikuporter do it for you with `hp -B` and `hp -e`, don't you ;)